video/out/hwdec/dmabuf_interop_gl.c

```diff
@@ -16,14 +16,33 @@
  */
 
 #include "config.h"
-#include "hwdec_vaapi.h"
+#include "dmabuf_interop.h"
 
 #include <EGL/egl.h>
 #include "video/out/opengl/ra_gl.h"
 
 typedef void* GLeglImageOES;
 typedef void *EGLImageKHR;
 
+// Redefine libdrm constants if necessary because the header location is not
+// consistent across distros, and so not worth messing with.
```

Comment:
Isn't that something only the build system should care about?
```c
#include <drm_fourcc.h>
```
should just work if libdrm was picked up right, since the packageconfig file will be used to get the include dir. If the PC doesn't lead us to the right headers then that's a packaging bug in the distro.